examples/restaurant_demo/README.md

```diff
@@ -0,0 +1,279 @@
+
+## Sum-up revenue per day of the week
+
+The previous example demonstrates how the contributed partitions are bounded.
+Now, you will demonstrate how individual contributions are clamped. Imagine Alice
+decides to calculate the sum of the restaurant revenue per week day in a
+differentially private way. For this she needs to sum up the visitors’ daily
+spending at the restaurant. For simplicity, let’s assume a visitor enters the
+restaurant at most once a day but multiple times a week.
+
+Visit data for a week is stored in the `week_data.csv` file.
+
+The results are illustrated in the image below.
+
+![Daily sums](img/sums_per_day.png)
+
+The code below uses `BoundedSum` to calculate the differentially private sums of
+the visitors' spendings for a single day.
+
+```
+
+# Cap the maximum number of visiting days at 4 per each visitor (any number above will not be taken into account)
+SUM_MAX_CONTRIBUTED_DAYS = 4
+
+# Expected minimum amount of money (in Euros) to be spent by a visitor per a single visit
+MIN_EUROS_SPENT = 0
+
+# Expected maximum amount of money (in Euros) to be spent by a visitor per a single visit
+MAX_EUROS_SPENT_1 = 50
+MAX_EUROS_SPENT_2 = 65
+
+LN_3 = math.log(3)
+
+  # Use the default epsilon value if it is not given as an argument
+        if not epsilon:
+            x = BoundedSum(
+                self._epsilon,
+                MIN_EUROS_SPENT,
+                MAX_EUROS_SPENT_1,
+                l0_sensitivity=SUM_MAX_CONTRIBUTED_DAYS,
+            )
+        else:
+            x = BoundedSum(
+                epsilon,
+                MIN_EUROS_SPENT,
+                MAX_EUROS_SPENT_1,
+                l0_sensitivity=SUM_MAX_CONTRIBUTED_DAYS,
+            )
+  // Calculate DP result.
+  result = x.result();
+```
+
+### Sum-up revenue per day of the week with preaggregation
+
+The usage of `SUM_MAX_CONTRIBUTED_DAYS` in `BoundedSum` is similar to its usage
+in `Count (COUNT_MAX_CONTRIBUTED_DAYS)`, which is explained in the previous example. This section focuses on
+the *lower* and *upper* bounds. The parameters `MIN_EUROS_SPENT` and `MAX_EUROS_SPENT_1` of
+`BoundedSum` define the *contribution caps*. Every input value will be
+automatically clamped to the specified bounds. This is needed for calculating
+the sensitivity of the aggregation, and to scale the noise that will be added to
+the sum accordingly.
+
+**Choosing bounds**.
+
+The min and max bounds affect the utility of the sum in two potentially
+opposing ways: reducing the added noise, and preserving the utility. On the one
+hand, the added noise is proportional to the maximum of the absolute values of
+the bounds. Thus, the closer the bounds are to zero, the less noise is added. On
+the other hand, setting the min and max bound close to zero may mean that
+the input values are clamped more aggressively, which can decrease utility as
+well.
+
+Let's set the `upper` bound of`BoundedSum` to 65 to reflect the approximate
```

Comment:
Line 253 - could you pls change to 

"The `upper` bound of`BoundedSum` to 65 to reflect the...." 

thank you